app/Http/Controllers/API/TraderController.php

```diff
@@ -152,9 +151,11 @@ public function showVoucherHistory(Trader $trader)
      */
     public function emailVoucherHistory(Request $request, Trader $trader)
     {
+        $vouchers = $trader->vouchersConfirmed;
+        $title = 'A report containing voucher history.';
         // Request date string as dd-mm-yyyy
         $date = $request->submission_date ? $request->submission_date : null;
```

Comment:
We target php7 and psr2 seems to permit conractions like this if they fit on one line; unless someone's codesniffer/ide plugins throws a fit, go a ahead. 

This line is fine how it is, though.